src/DynamoCore/Models/DynamoModel.cs

```diff
@@ -1482,8 +1482,11 @@ public bool OpenCustomNodeWorkspace(Guid guid)
             {
                 if (!Workspaces.OfType<CustomNodeWorkspaceModel>().Contains(customNodeWorkspace))
                     AddWorkspace(customNodeWorkspace);
+
                 CurrentWorkspace = customNodeWorkspace;
+                return true;
             }
+
             return false;
         }
```

Comment:
Strictly speaking this isn't part of the fix, but it was too glaring that I couldn't let it go, so here it is...